### PR TITLE
[FEAT] - Sidebar Mobile

### PR DESCRIPTION
**Pull request by adzaky**
https://github.com/user-attachments/assets/bf55115a-b17c-4e1d-9703-c53792974e73

### CODE REVIEW

File: src/components/common/sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const MenuOverlay = () => (
+    <div
+      className="fixed top-0 bottom-0 left-0 right-0 z-30 flex md:hidden bg-black/40"
+      onClick={() => {
+        setShowSidebar(false);
+      }}
+    />
+  );
+
+  const MenuBarMobile = () => (
```
Comment (vickyadrii) [inline src/components/common/sidebar.tsx]: It's better to move this code to a new component file. maybe you might consider creating a navbar folder and adding MenuItem, MenuOverlay, and MenuBarMobile components to it. cc: @adzaky

Comment (adzaky) [reply src/components/common/sidebar.tsx]: image
Structure looks fine now? @vickyadrii

Comment (adzaky) [reply src/components/common/sidebar.tsx]: image

Or this one?

Comment (vickyadrii) [reply src/components/common/sidebar.tsx]: looks good! make the folder name to lowercase. approved @adzaky

**Review by vickyadrii — APPROVED** [review]

**Pull request merged**